ethereum/README.md

```diff
@@ -2,61 +2,101 @@
 
```

Comment:
Can we rather use the title "Ethereum Contracts"?

Reply:
Updated in https://github.com/Snowfork/polkadot-ethereum/pull/107/commits/5d02b9d9f0e7f00a7a8082af1a854a096165dee6